images/Dockerfile

```diff
@@ -6,11 +6,11 @@ FROM ${BASE_IMAGE} AS base
 ARG VARIANT=core
 ARG MODEL=generic
 ARG TRUSTED_BOOT=false
-ARG KUBERNETES_PROVIDER=k3s
+ARG KUBERNETES_DISTRO=k3s
```

Comment:
I think DISTRO is a better name because providers we have the kairos provider which can handle multiple distros, there's also a k3s provider which happens to also have the k3s kubernetes distro but they are different things